fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/ambassador/AbstractSumoAmbassador.java

```diff
@@ -726,7 +741,7 @@ private synchronized void receiveInteraction(VehicleLaneChange vehicleLaneChange
                     return;
             }
             bridge.getVehicleControl().changeLane(vehicleLaneChange.getVehicleId(), targetLaneId,
-                    convertDuration(vehicleLaneChange.getDuration()));
+                    nsToMs(vehicleLaneChange.getDuration()));
```

Comment:
We should change the API commands here to pass either double (seconds) or long (ns) instead of milliseconds. So we get rid of `ns2ms`. SUMO works with seconds for a long time already. If not in this PR, then please add an issue for this.